torch/csrc/jit/lite_interpreter/bytecode.cpp

```diff
@@ -0,0 +1,166 @@
+#include "bytecode.h"
```

Comment:
Have interpreterState equavalent to hold the registers. Mirror the layout in interpreter.cpp

Reply:
Add backward compatibility comments here. Backward compatibility is not supported currently. In future we need to revisit it if there's need to support backward compatibility.